simapp/export.go

```diff
@@ -18,17 +18,28 @@ import (
 func (app *SimApp) ExportAppStateAndValidators(
 	forZeroHeight bool, jailAllowedAddrs []string,
 ) (servertypes.ExportedApp, error) {
+	return app.ExportAppStateAndValidatorsAt(forZeroHeight, jailAllowedAddrs, 0)
+}
+
+func (app *SimApp) ExportAppStateAndValidatorsAt(
```

Comment:
Let's add doc comments. Especially explaining that with height=0 the latest height will e used.